src/FSharp.Control.TaskSeq/Utils.fsi

```diff
@@ -80,25 +83,22 @@ module Task =
     /// Bind a Task<'T>
     val inline bind: binder: ('T -> #Task<'U>) -> task: Task<'T> -> Task<'U>
 
-    /// Create a task from a value
-    val inline fromResult: value: 'U -> Task<'U>
-
 module Async =
 
     /// Convert an Task<'T> into an Async<'T>
     val inline ofTask: task: Task<'T> -> Async<'T>
 
-    /// Convert a unit-task into an Async<unit>
+    /// Convert a non-generic Task into an Async<unit>
```

Comment:
Note that the term "unit-task" is used in some other libraries. I'm also not really a fan of specifying something by mentioning the absence of something (but I also don't have a good alternative).

Reply:
what are these libraries?
I personally intuitively got what it meant and liked it when I first saw it.
But happy to flip all the 'non-generic X` to `unit-X` in this PR; my main desire is consistency...

Reply:
I forgot what lib it was, but I know I didn't invent the term. I did a quick search, but searching for `unit`  and ` task` in F# repos (and GH not allowing string search) is like a needle in a haystack.

I'm fine either way. Though there's something to say that both `Task`  and `Task<unit>` could be consisered unit-tasks (or side-effect tasks or whatevs)l.

Reply:
Your choice ;)

Reply:
I think the non-generic task terminology edges it for me
- its explicit and unambiguous - saying what it does and when you need it
- does not lean on FP terms
- side effecting and/or the distinction between a `Task<unit>` or something with the same intent that happens to return `Task` for techical reasons are a not the point
- even if this was about a `Task<unit>` that had been upcasted to `Task` (or someone was ignoring the result intentionally by that mechanism), the original was a unit task
So I'll make it consistently 'non-generic' for now, but won't object/when if/when you flip all the cases to something else consistent

Reply:
I note you have some `<returns>A <see cref="unit" /> <see cref="task" />.</returns>` usage in other places - I think it's clear and accurate there (even if I think I'd just say `Task<unit>` and be done with it had I not seen that way of writing it) e.g. https://github.com/fsprojects/FSharp.Control.TaskSeq/blob/e76f9212934f8a5dd367e3fa42fb31ebb44b4bb6/src/FSharp.Control.TaskSeq/TaskSeq.fsi#L521

I'll let you Resolve this or pull rank

Reply:
I'll check it. For now let's leave it in as you wrote it.